src/main/java/seedu/storemando/model/item/Item.java

```diff
@@ -123,4 +123,20 @@ public String toString() {
         return builder.toString();
     }
 
+    public int compareByItemName(Item anotherItem) {
```

Comment:
*Carry on from quantityComparator and ExpiryDateComparator*

All this method can be removed and replace with a method like quantityPriorityComparison and expiryDateComparison. It would somewhat be like a compareTo method in the Comparable interface.